src/TestUtils/src/TestUtils.Appium.UITests/AppiumUITestApp.cs

```diff
@@ -666,7 +667,8 @@ public void Tap(Func<AppQuery, AppWebQuery> query)
 
 		public void TapCoordinates(float x, float y)
 		{
-			throw new NotImplementedException();
+			var ta = new TouchAction(_driver);
```

Comment:
TouchAction is deprecated, look into ActionSequence, if you run into issues with that on mac/ios you could use the ExecuteScript functionality. Ping me if you want to talk about it.
https://github.com/appium/appium/issues/16009